app/services/hyrax/custom_queries/navigators/child_filesets_navigator.rb

```diff
@@ -11,7 +11,8 @@ module Navigators
       class ChildFilesetsNavigator
```

Comment:
I did not refactor the class name, but am open to doing that.

Reply:
It looks like this navigator is the only class named that way.  I think it would be good to cleanup.
How hard would it be to deprecate it and change it?